lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -124,9 +131,14 @@ def call(params)
             :role => row[role_column] || row[role_column.to_sym],
             :sso_provider => sso_provider || row[sso_provider_column] || row[sso_provider_column.to_sym],
             :authentication_modes => modes,
-            user_group: row[user_groups_column] && row[user_groups_column].split(',').map(&:strip),
+            :user_group => row[user_groups_column] && row[user_groups_column].split(',').map(&:strip),
             :pid => multiple_projects_column.nil? ? nil : (row[multiple_projects_column] || row[multiple_projects_column.to_sym]),
-            :language => row[language_column] || row[language_column.to_sym]
+            :language => row[language_column] || row[language_column.to_sym],
+            :company => row[company_column] || row[company_column.to_sym],
+            :position => row[position_column] || row[position_column.to_sym],
+            :country => row[country_column] || row[country_column.to_sym],
+            :phone => row[phone_column] || row[phone_column.to_sym],
+            :ip_whitelist => ip_whitelist
           }.compact
```

Comment:
Please write a unit test that checks the result of the action.

Reply:
there: https://ms-gerrit.intgdc.com/#/c/740/